src/cache.spec.ts

```diff
@@ -1,11 +1,12 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
I think `any` for tests is acceptable, although we could put some effort and improve the types there too.

Thoughts?

Reply:
yeah I don't think it's an issue, be we can address in the future if we want to